File: 0075-sort-colors/0075-sort-colors.c
void sortColors(int* nums, int numsSize) {
    for(int i=0;i<numsSize;i++){
        for(int j=i+1;j<numsSize;j++){
            if(nums[i]>=nums[j]){
                int temp=nums[j];
                nums[j]=nums[i];
                nums[i]=temp;
            }
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
     int n = nums.length;
     int zero =0,one =0;
     for(int i=0;i<n;i++){
        if(nums[i]==0)
        zero++;
        else if(nums[i]==1)
        one++;
     }
     for(int i=0;i<zero;i++){
        nums[i]=0;
     }
     for(int i=zero;i<zero+one;i++){
        nums[i]=1;
     }
     for(int i=zero+one;i<n;i++){
        nums[i]=2;
     }

    }
}